apps/angular/crud/src/app/ui/list-item/list-item.component.ts

```diff
@@ -0,0 +1,30 @@
+import { NgTemplateOutlet } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ContentChild,
+  TemplateRef,
+  input,
+  output,
+} from '@angular/core';
+import { LetDirective } from '@ngrx/component';
+import { LoadingStatePipe } from '../../shared/pipe/loading-state.pipe';
+import { LoadingState } from '../../shared/state/loading.feature';
+import { LoaderComponent } from '../loader/loader.component';
+
+@Component({
+  selector: 'app-list-item',
+  templateUrl: './list-item.component.html',
+  styleUrls: ['./list-item.component.scss'],
+  standalone: true,
+  imports: [NgTemplateOutlet, LoaderComponent, LoadingStatePipe, LetDirective],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ListItemComponent {
+  loadingState = input.required<LoadingState>();
+  onUpdate = output();
+  onDelete = output();
+
+  @ContentChild('itemContentTemplate')
```

Comment:
Not a big deal, but ...

> it's always better to reference a directive than a magic string
[https://dev.to/this-is-angular/ngtemplateoutlet-typed-checked-with-contentchild-2phe](https://dev.to/this-is-angular/ngtemplateoutlet-typed-checked-with-contentchild-2phe)

Reply:
Just updated the PR removing the magic string